confirmer/confirmer.go

```diff
@@ -151,8 +151,21 @@ func (confirmer *Confirmer) lockTxConfirmed(ctx context.Context, transaction tx.
 			Index: input.Txindex,
 		})
 		if err != nil {
-			confirmer.options.Logger.Errorf("[confirmer] cannot get output for utxo tx=%v (%v): %v", input.Txid.String(), transaction.Selector.String(), err)
-			return false
+			// try the reversed txid
```

Comment:
Can you remind me why this is necessary here?

Reply:
Let me double check - at a point the utxos were being provided in both reversed and un-reversed formats, but in retrospect that should not be the case, will test again

Reply:
Sweet.